.github/ISSUE_TEMPLATE/general-question.yml

```diff
@@ -0,0 +1,19 @@
+name: General Question
```

Comment:
An alternative to questions could be using [discussions](https://docs.github.com/en/discussions)? Just an idea.

Reply:
Yes I think we should remove the `General Question` option and redirect to discussions or something.

Reply:
Damn, I forgot about this feature 😅 
Yeap. Let's remove the General Question and have discussions